fn_ansible_tower/apikey_permissions.txt

```diff
@@ -10,17 +10,17 @@ read_function
 #
 ## incident permissions
 #create_incident
-#read_all_incidents
+read_all_incidents
```

Comment:
why is this needed?

Reply:
package uses data table to update the results. so felt its needed.